polaris-react/README.md

```diff
@@ -75,16 +72,13 @@ If React doesn’t make sense for your application, you can use a CSS-only versi
 
 ### Usage
 
-1.  Include the CSS in your HTML. We suggest copying the styles file into your own project, but you may also use it directly:
+1. Include the CSS in your HTML. Copying the latest version of the [styles file](https://unpkg.com/@shopify/polaris@latest/build/esm/styles.css) into your own project. This will need to be updated with future releases.
```

Comment:
```suggestion
1. Include the CSS in your HTML. We suggest copying the latest version of the [styles file](https://unpkg.com/@shopify/polaris@latest/build/esm/styles.css) into your own project. This will need to be updated with future releases.
```